src/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -246,9 +246,9 @@ void f()
         }
 
         [Theory]
-        [InlineData(Language.CSharp)]
-        [InlineData(Language.FSharp)]
-        public async Task it_returns_diagnostics(Language language)
+        [InlineData(Language.CSharp, "CS0103", "The name 'aaaadd' does not exist in the current context", "(1,1): error CS0103: The name 'aaaadd' does not exist in the current context")]
+        [InlineData(Language.FSharp, "FS0039", "The value or constructor 'aaaadd' is not defined.", "input.fsx (1,1)-(1,7) typecheck error The value or constructor 'aaaadd' is not defined.")]
+        public async Task it_returns_diagnostics_on_command_failed(Language language, string code, string diagnosticMessage, string errorMessage)
```

Comment:
This test should be "when_code_contains_compile_time_error_diagniostics_are_produced"